website/docs/reference/declaring-properties.md

```diff
@@ -96,6 +96,25 @@ models:
 
 </File>
 
+You can also use Jinja to conditionally configure the schema associated with sources. Anything that can be computed in Jinja can be used to determine what schema to look for a table in at runtime.
+
+<File name='models/jaffle_shop_with_dynamic_schema.yml'>
+
+```yml
+version: 2
+
+sources:
+  - name: raw_jaffle_shop
+    schema: {{ 'core_production' if target.name == 'prod' else 'core_staging' }}
```

Comment:
I _think_ this might need more quotes around it!
```suggestion
    schema: "{{ 'core_production' if target.name == 'prod' else 'core_staging' }}"
```